scripts/globals/utils.lua

```diff
@@ -439,14 +439,11 @@ local systemStrengthTable =
     [xi.eco.ARCANA  ] = { [xi.eco.UNDEAD  ] = 1, },
     [xi.eco.DRAGON  ] = { [xi.eco.DEMON   ] = 1, },
     [xi.eco.DEMON   ] = { [xi.eco.DRAGON  ] = 1, },
-    [xi.eco.LUMORIAN] = { [xi.eco.LUMINION] = 1, },
-    [xi.eco.LUMINION] = { [xi.eco.LUMORIAN] = 1, },
+    [xi.eco.LUMINIAN] = { [xi.eco.LUMINION] = 1, },
+    [xi.eco.LUMINION] = { [xi.eco.LUMINIAN] = 1, },
 }
 
-function utils.getSystemStrengthBonus(attacker, defender)
-    local attackerSystem = attacker:getSystem()
-    local defenderSystem = defender:getSystem()
-
+function utils.getSystemStrengthBonus(attackerSystem, defenderSystem)
```

Comment:
Is this function only used by blue spells?

If not, any other system that uses it will still feed it caster and target and fail to work properly

Reply:
So I went ahead and made a mass search

This function is indeed used elsewhere

So please rever this changes, so the function handles the attacker and defender system

Reply:
EDIT: Ugh, nevermind, i screwed up